berth_reservations/tests/utils.py

```diff
@@ -1,9 +1,53 @@
 import json
 
-from django.test import Client
+from django.contrib.auth.models import AnonymousUser
+from django.test import Client, RequestFactory
+from graphene.test import Client as GrapheneClient
 
+from berth_reservations.new_schema import new_schema
+from berth_reservations.schema import schema
 
-class GraphQLTestClient(Client):
+
+class ApiClient(GrapheneClient):
+    def execute(self, *args, **kwargs):
+        """
+        Custom wrapper on the execute method, that allows passing
+        "input_variables" as a keyword argument, which get passed
+        to kwargs["variables"]["input"] to comply with the relay
+        spec for mutations.
+        """
+
+        input_variables = kwargs.pop("input_variables", {})
```

Comment:
Is there any specific reason to name the kwarg `input_variables` instead of just `input` or `variables`?

Reply:
I was thinking how to name it, and thought that:
1. `variables` should be the root variables dictionary as per GQL specs
2. `input` could work but it could seem a bit unclear what kind of input

we could go with the `input`, if it seems clear enough

Reply:
I kinda liked how it was handled in the old `GraphQLDjangoTestClient` by passing `variables`, it felt clearer (or maybe I just got used to it 🤷‍♂ ).

But I would suggest the "standard" that we already have on the [`mutations`](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/resources/schema.py#L343) and keep it as `input`
```python
def mutate_and_get_payload(cls, root, info, **input):
    pass
```

Thoughts?

Reply:
yeah, I faced problems with the old way, since not all of the mutations follow relay spec. and it felt kinda hacky that we were forcing variables to be wrapped in an `input`. the main problem was that I needed to pass simply `variables` to the API, don't remember the use case though...
anyway, having an option to either pass `variables` dict or `input` dict, which wraps them in `{"input": variables}` is a decent solution. let's do that 👍

Reply:
Yeah, coercing `variables` always to `{"variables": {"input": variables}}` could work but definitely flawed.

Like you say, if we have both options to pass either ```variables -> {"variables": variables}``` or ```input -> {"variables": {"input": input}}``` sounds more reasonable.